ui/admin/tests/unit/controllers/scopes/scope/host-catalogs/index-test.js

```diff
@@ -0,0 +1,31 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'admin/tests/helpers';
+import { waitUntil } from '@ember/test-helpers';
+
+module(
+  'Unit | Controller | scopes/scope/host-catalogs/index',
+  function (hooks) {
+    setupTest(hooks);
+
+    let controller;
+
+    hooks.beforeEach(function () {
+      controller = this.owner.lookup(
+        'controller:scopes/scope/host-catalogs/index',
+      );
+    });
+
+    test('it exists', function (assert) {
+      assert.ok(controller);
+    });
+
+    test('handleSearchInput action sets expected values correctly', async function (assert) {
+      const searchValue = 'test';
+      controller.handleSearchInput({ target: { value: searchValue } });
+      await waitUntil(() => controller.search === searchValue);
+
+      assert.strictEqual(controller.page, 1);
+      assert.strictEqual(controller.search, searchValue);
+    });
+  },
```

Comment:
Could we add a unit test for `applyFilter` as well?

Reply:
There are no filters

Reply:
Apologies, I thought I saw the `applyFilter` action in the controller.